src/styles.scss

```diff
@@ -239,5 +239,25 @@ body {
     height: 50px;
     padding: 3px;
   }
+  
+  .space-container {
+    margin: 64px 30px;
+  }
+
+  .mat-column-select {

```

Comment:
These column sizes are actually specific to those components, so should stay as part of the component styles.